### PR TITLE
chore(bench): make compression benchmarks available for database

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -893,6 +893,12 @@ bench_integer_gpu: install_rs_check_toolchain
 	--bench integer-bench \
 	--features=$(TARGET_ARCH_FEATURE),integer,gpu,internal-keycache,nightly-avx512 -p $(TFHE_SPEC) --
 
+.PHONY: bench_integer_compression # Run benchmarks for unsigned integer compression
+bench_integer_compression: install_rs_check_toolchain
+	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_CHECK_TOOLCHAIN) bench \
+	--bench	glwe_packing_compression-integer-bench \
+	--features=$(TARGET_ARCH_FEATURE),integer,internal-keycache -p $(TFHE_SPEC) --
```
Comment (IceTDrinker) [inline Makefile:900]: AVX512 ?

File: tfhe/benches/integer/glwe_packing_compression.rs [hunk 2]
```diff
@@ -46,26 +48,38 @@ fn cpu_glwe_packing(c: &mut Criterion) {
...
 
                 _ = black_box(compressed);
             })
         });
 
+        write_to_json::<u64, _>(
+            &bench_id,
+            comp_param,
+            "COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64",
+            "unpack",
```
Comment (IceTDrinker) [inline tfhe/benches/integer/glwe_packing_compression.rs:73]: pack or unpack ?

File: tfhe/benches/integer/glwe_packing_compression.rs [hunk 3]
```diff
@@ -144,9 +159,19 @@ fn gpu_glwe_packing(c: &mut Criterion) {
             })
         });
 
+        write_to_json::<u64, _>(
+            &bench_id,
+            comp_param,
+            "COMP_PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64",
```
Comment (IceTDrinker) [inline tfhe/benches/integer/glwe_packing_compression.rs:165]: parameters are not named ?

File: tfhe/benches/integer/glwe_packing_compression.rs (hunk 2 above)
Comment (IceTDrinker) [reply tfhe/benches/integer/glwe_packing_compression.rs:73]: yes but I think this result is for the pack not unpack ?

Comment (soonum) [reply tfhe/benches/integer/glwe_packing_compression.rs:62]: You're right.

**Review by IceTDrinker — APPROVED** [review]
Looks good to me ! Thanks !

**Comment by soonum** [comment]
Forgot to add the benchmarks recipe call in the GPU workflow.

**Review by IceTDrinker — APPROVED** [review]
🔥

**Pull request merged**